test/packages/time_series/data_stream/example/manifest.yml

```diff
@@ -9,3 +9,9 @@ streams:
         type: text
         title: Period
         default: 10s
+
+elasticsearch:
```

Comment:
Hm.. just a test file? double-checking :)

Reply:
Well, I was going to TDD this and found that nothing was needed :D

Reply:
Ah ok, now it makes sense :)